index.js

```diff
@@ -214,23 +214,24 @@ SteamTradeOffers.prototype.cancelOffer = function(options, callback) {
   });
 };
 
-SteamTradeOffers.prototype.acceptOffer = function(options, callback) {
+SteamTradeOffers.prototype.acceptOffer = function(offer, callback) {
```

Comment:
The convention of this module is to have an options object as the first argument and params must be camelcase. Options may contain anything not just offer details which is more future-proof than what you are currently proposing (and I don't see how there will be no need to change the library code if another similar issue arises).